roles/zooplus/tasks/globalprotect.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Check if globalprotect client is installed
+  stat:
+    path: /usr/bin/globalprotect
+  register: globalprotect_existence
+
+- name: Copy globalprotect package
+  copy:
+    src: files/{{ globalprotect_package }}
+    dest: /home/{{ user }}/SW/{{ globalprotect_package }}
+  when: not globalprotect_existence.stat.exists
+
+- name: Install globalprotect
+  command: "dpkg -i /home/{{ user }}/SW/{{ globalprotect_package }}"
+  register: globalprotect_installation_check
+  when: not globalprotect_existence.stat.exists
+
+- name: Check globalprotect installation
```

Comment:
I personally don't this. I trust apt and her error handling.

Reply:
only if I had a correct globalprotect package than can be installed via apt...

i wish....